cppcache/integration/test/RegisterKeysTest.cpp

```diff
@@ -46,9 +46,11 @@ Cache createTestCache() {
       .create();
 }
 
-std::shared_ptr<Region> setupCachingProxyRegion(Cache& cache) {
+std::shared_ptr<Region> setupCachingProxyRegion(Cache& cache,
```

Comment:
Prefer an overload over a default parameter. This obfuscates the code as documentation and the meaning of all the prior tests. You're explicitly stating concurrency checks are not to be enabled. Why is that an important condition for the other tests? I presume what you wanted was the ability to enable checks for your one test.

Reply:
You have a good point indeed. Noted :)